### PR TITLE
INTELLIJ-144 Liferay-intellij-plugin 1.8.0 installation failed on Intellij IC 2020.2.2

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -78,6 +78,12 @@ dependencies {
 	testCompile group: "junit", name: "junit", version: "4.12"
 }
 
+jar {
```
Comment (gamerson) [inline build.gradle:81]: hey @simonjhy  what does this do?

Comment (gamerson) [reply build.gradle:81]: This seems like it is copying too much into the jar.  The jar grew over 7 MBs with this change.  What is missing exactly that you were trying to include?

**Review by gamerson — CHANGES_REQUESTED** [review]
Hey @simonjhy  can you take a look at the jar task changes you made and only include the missing files explicitly instead of pulling in everything which contains seemingly unrelated things?  Thanks

File: build.gradle (hunk 1 above)
Comment (simonjhy) [reply build.gradle:81]: ok

Comment (simonjhy) [reply build.gradle:81]: commons-configuration.jar it seems intellij remove its support. We did not have this problem in last version intellij.

**Pull request closed without merging**